.gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
 dist
 node_modules
+.idea
```

Comment:
Hi @bentatum - My apologies for the delay. I use phpStorm (Based on the IDEA IDE from Jet Brains) and they store project files in the hidden `.idea` directory. I'm happy to wipe it out if you'd like.